src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -273,6 +287,8 @@ function Content(props: {
         accessCodes={accessCodes}
         filter={filter}
         onFilterSelect={setFilter}
+        errorFilter={errorFilter}
```

Comment:
The main case I want to check on is if the health bar / list elements always have the same filter logic as the detail view. Same for device table. 

I'm wondering if the health bar here and whatever component renders an issue icon for a row needs to have the same filter logic as AccessCodeDetail. It's not super DRY as there is nothing ensuing the `AccessCodeDetail` has the same filter, but we should still be able to make it internally consistent.

Reply:
Gotcha, okay—I will add this in now.